src/testing.ts

```diff
@@ -103,7 +103,7 @@ const expectedResults = new Map<string, ResultType>([
   ['example_contracts/state_variables/arrays', 'CairoCompileFailed'],
   ['example_contracts/state_variables/mappings', 'Success'],
   ['example_contracts/state_variables/structs', 'CairoCompileFailed'],
-  ['example_contracts/state_variables/misc', 'NotSupportedYet'],
+  ['example_contracts/state_variables/misc', 'TranspilationFailed'],
```

Comment:
If it's possible to turn this into a NotSupportedYet that would be appreciated, I prefer not to have any TranspilationFailed merged in. It should also have a comment as to why it fails